### PR TITLE
Larryebaum patch 2

### PR DESCRIPTION
**Pull request by larryebaum**
## PR Objective

- [ ] Fixing inaccurate docs
- [ ] Making some docs easier to understand
- [ ] Adding/updating docs for new feature
- [ ] Changing behavior or layout of website

### CODE REVIEW

**Comment by larryebaum** [comment]
Hi team. I’m submitting another PR for a docs edit (OKTA SAML). Usually I just make simple text edits, this one has screenshots (uploaded in same PR) but they don’t seem to appear on my preview — prob did something wrong. Hoping second eyes and someone help me with what I may have done wrong so I can redo/learn.

File: content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ Follow these steps to configure Okta as the identity provider (IdP) for Terrafor
...
     - **Value:** An [Okta expression](https://developer.okta.com/reference/okta_expression_language/) that will evaluate to a boolean: `true` for every user who should have site admin permissions, but `false` for any users who should **not** have site admin permissions. The exact expression depends on the user properties you use to manage admin permissions.
 7. Preview the SAML response and make sure it matches your expectations.
-8. Finish configuring the SAML app in Okta, then copy the provided endpoint URLs and certificate to your Terraform Enterprise SAML settings at `https://<TFE_HOSTNAME>/app/admin/saml`. TFE requires a single sign-on URL, a single log-out URL, and a PEM (base64) encoded X.509 certificate.
+8. Finish configuring the SAML app in Okta, then copy the provided endpoint URLs and certificate to your Terraform Enterprise SAML settings at `https://<TFE_HOSTNAME>/app/admin/saml` (example below). TFE requires a single sign-on URL, a single log-out URL, and a PEM (base64) encoded X.509 certificate.
+      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (1 of 2), with populated example data.](./images/sso-okta-screen1.png)
```
Comment (sudomateo) [inline content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md:43]: The image you added is named `sso-tfe-okta-screen1.png`, not `sso-okta-screen1.png`.

```suggestion
      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (1 of 2), with populated example data.](./images/sso-tfe-okta-screen1.png)
```

File: content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md [hunk 2]
```diff
@@ -39,7 +39,9 @@ Follow these steps to configure Okta as the identity provider (IdP) for Terrafor
...
+      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (2 of 2), with populated example data.](./images/sso-okta-screen2.png)
```
Comment (sudomateo) [inline content/source/docs/enterprise/saml/identity-provider-configuration-okta.html.md:44]: The image you added is named `sso-tfe-okta-screen2.png`, not `sso-okta-screen2.png`.

```suggestion
      ![Screenshot: TFE "Identity Provider Configuration" page for SAML (2 of 2), with populated example data.](./images/sso-tfe-okta-screen2.png)
```

**Comment by larryebaum** [comment]
@sudomateo Thanks for that... a sloppy error on my side. I'll make a note of the link/instructs you shared and hope to remember in future (usually rushing these edits in while on calls).

**Pull request merged**